polygon/bor/bordb/prune.go

```diff
@@ -30,6 +30,7 @@ import (
 // doesn't change sequences of kv.EthTx
 // doesn't delete Receipts, Senders, Canonical markers, TotalDifficulty
 func PruneBorBlocks(tx kv.RwTx, blockTo uint64, blocksDeleteLimit int, SpanIdAt func(number uint64) uint64) (deleted int, err error) {
```

Comment:
Can you explain the return value also? Is `deleted` the total number of objects deleted?